specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json

```diff
@@ -364,7 +364,7 @@
           "description": "(Optional) Collection of participants invited to the room.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/RoomParticipant"
+            "$ref": "#/definitions/InvitedRoomParticipant"
```

Comment:
I wonder if naming CreateRoomParticipant is more accurate and consistent with current naming convention used in other models.

Reply:
Resolve offline - keeping as is for now and waiting for feedback from DevEx team